refinery/tool_manager/models.py

```diff
@@ -255,12 +259,16 @@ def launch(self):
                         self.file_relationships
                     )
                 },
-                manager=get_django_docker_engine_manager()
+                extra_directories=ast.literal_eval(
+                    self.tool_definition.extra_directories
+                )
             )
             try:
-                container.run()
+                DockerClientWrapper().run(container)
             except APIError as e:
                 return HttpResponseServerError(content=e)
+            except AssertionError as e:
+                return HttpResponseBadRequest(content=e)
```

Comment:
Yeah, good call. I just did this to handle for the assertion we added for having absolute filesystem paths. 
Catching something less specific would probably be better here.